src/pages/landing/ebooks/components/_get-ebook.tsx

```diff
@@ -343,7 +343,7 @@ const GetEbook = ({ color = 'var(--color-white)', ebook_utm_code, onSubmit }: Ge
                             components={[
                                 <StyledLocalizedLink
                                     key={0}
-                                    to="/tnc/security-and-privacy.pdf"
+                                    href="/tnc/security-and-privacy.pdf"
```

Comment:
Probably we have to make `is_eu` condition because we have different rules for them. Left comment in card, let's wait QA decision